src/ACEtk/block/TabulatedOutgoingEnergyDistribution/src/ctor.hpp

```diff
@@ -0,0 +1,42 @@
+TabulatedOutgoingEnergyDistribution() = default;
```

Comment:
[clang-tidy - recommended]
All constructors are defined, but there is no destructor.